pkg/infra/main.go

```diff
@@ -26,10 +29,32 @@ const monsterHealth = 100
 const monsterAttack = 240
 
 func main() {
+	// define flags
+	configFile := flag.String("c", "config.json", "configuration file")
```

Comment:
Working on setting up config params as part of a seperate PR with @neeldug. Can we remove this line from this specific PR and introduce it again later if required?

Reply:
I saw your PR, thought this might be a helpful addon for #5. One use case might be, we provide an example/default config while each team can create and use their own configs in their test runs by specifying different paths in their scripts without modifying the main function.

I will remove it if you worry about conflicts.